addons/website/static/src/systray_items/website_switcher.xml

```diff
@@ -11,12 +11,17 @@
                 <i class="fa fa-globe"/>
             </div>
         </t>
+        <t t-set="tooltipValue">This website does not have a domain configured.</t>
         <t t-foreach="getElements()" t-as="element" t-key="element_index">
             <DropdownItem
-                t-esc="element.name"
                 onSelected="element.callback"
                 class="element.class"
-            />
+                dataset="{'tooltip': tooltipValue, 'tooltipPosition': 'left'}">
```

Comment:
This will add the tooltip on website which also have a website set
![image](https://github.com/odoo/odoo/assets/30048408/821ff357-6a5a-4c6d-a20c-a206a39b21d7)
That's what I was [concerned](https://github.com/odoo/odoo/pull/128979#discussion_r1274726370) about when I said "Ignore if too complicated"
`t-att-dataset` conditioned on domain set or not maybe?

Reply:
Oh yeah that was dumb of me! I made [this diff](https://github.com/odoo/odoo/compare/08eccce7c6e18367e8b022303082383c835cda76..3920f8c34477235908bc1db9a168abf50da1a0f8) which should fix the problem!